participants/Martínez_Fernández/pr_tutorial/simple_functions.py

```diff
@@ -10,3 +10,14 @@ def factorial(value):
         return 1
     else:
         return value * factorial(value - 1)
+
+def is_prime(value):
+    if value > 2:
```

Comment:
There is probably a way to avoid going through specific cases, but let's not worry about it for now.